Kickstarter-iOS/Features/SortFilter/FilterCategory/FilterCategoryViewModel.swift

```diff
@@ -46,18 +51,24 @@ class FilterCategoryViewModel<T: FilterCategory>: FilterCategoryViewModelType {
       .assign(to: &self.$canReset)
 
     self.selectedCategorySubject
+      .map { $0?.0 }
```

Comment:
nit: naming these instead of going with the short hand could make this easier to review in the future.